.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 install:
 - testing/install_test_dependencies.sh
+- echo "extension = mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
If you use the `services` directive, it will automatically activate the extension in `php.ini`, so this line is not necessary